docs/organizations/managing-security-and-risk.md

```diff
@@ -1,73 +1,129 @@
 
-## Exporting the security item list
+### Open findings history
+
+The **Open findings history** graph shows the open findings trends over the past three months, grouped by week and severity. It details the progression of your organization's risk and security posture over time and can, for example, help you understand if the right issues are being addressed.
+
+For a detailed view of the distribution on a specific week, hover over the graph.
+
+![Security and risk management open findings history panel](images/security-risk-management-overview-history-open.png)
+
+### Activity history
+
+The **Activity history** graph shows weekly counts of open and closed findings over the past three months, overlaid on the overall open findings trend. It complements the **Open findings history** graph with more information, such as the volume of findings addressed each week and a visual representation of the new/closed ratio.
+
+For a detailed view of the counts on a specific week, hover over the graph.
+
+![Security and risk management activity history panel](images/security-risk-management-overview-history-activity.png)
+
+### Top 10 high-risk repositories
+
+The **Top 10 high-risk repositories** list shows the repositories with the highest number of open findings, ordered by severity.
+
+!!! note
+    This panel may list fewer than ten repositories if there are fewer than ten repositories with open findings in the organization or if fewer than ten repositories are selected in the dropdown **Repository** filter.
+
+![Security and risk management high risk repositories panel](images/security-risk-management-overview-top-risk.png)
+
+### Top 10 common security categories
+
+The **Top 10 common security categories** list shows the most common security categories of open findings, ordered by count.
+
+To access the findings page with the corresponding filter applied, click on a category.
+
+![Security and risk management top categories panel](images/security-risk-management-overview-top-categories.png)
+
+## Findings {: id="item-list"}
+
+The **Security and risk management findings** page displays a filtered list of findings. By default, this list is sorted by status, and you can click the **First detected** column name to sort the findings by the detection date. Use this page to review and prioritize findings and track the progress of your security efforts.
+
+To access the findings page, access the [overview page](#dashboard) and click the **Findings** tab.
+
+![Security and risk management findings page](images/security-risk-management-findings.png)
+
+When viewing the findings, you can update the filtering criteria by clicking the **Severity**, **Status**, **Repository**, or **Security category** drop-downs above the list.
+
+To find out more about a finding in the list, click its **Details** column to navigate to the finding details on the source platform. Click **View all findings** to get back to the findings list.
```

Comment:
@nicklem I'm not sure how this works when the source platform is not Codacy... If the source is not Codacy, maybe the **View all findings** link doesn't apply, right?

**Side note:** Are we still presenting findings from other sources? Why don't we have that information on the list anymore, and we can't filter by source? 🤔

Reply:
Removing the reference to **View all findings** link, as it doesn't apply to all sources (and it's not really needed, the UX is self-explanatory).

```suggestion
To find out more about a finding in the list, click its **Details** column to navigate to the finding details on the source platform.
```